pennylane/optimize/__init__.py

```diff
@@ -39,4 +40,5 @@
     "QNGOptimizer",
     "RotosolveOptimizer",
     "RotoselectOptimizer",
+    "ShotAdaptiveOptimizer",
```

Comment:
It does not display in the "optimisers" section though? I think there we have a manual list!

Reply:
Oh do we? 🤦

Reply:
added!